_posts/en/newsletters/2023-09-06-newsletter.md

```diff
@@ -0,0 +1,139 @@
+  plus one or more keys held by one or more service providers who only
+  sign after authenticating the user in some way.  The user can bypass
+  the service provider if they need to, but the service provider makes
+  operations easier in most cases.
+
+    With scripted threshold signature schemes like 2-of-3
+    `OP_CHECKMULTISIG`, the service's public key must be associated with
+    the output being spent, so any service will be able to find the
+    transactions it signed by looking at onchain data, allowing it
+    accumulate data about its users.  Worse, all currently used protocols
+    we're aware of directly reveal user transactions to the service
+    provider before signing, allowing the service to refuse to sign
+    certain transactions.
+
+    As Farrow describes, FROST allows hiding the signed transaction from
+    the service at every step of the process, from generation of an
+    output script, to signing, to publication of the fully signed
+    transaction.  All the service will know is when it signed and any
+    data the user provided to authenticate themselves with the service.
+
+    The idea received some discussion on the mailing list.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test release candidates.*
+
+- [Libsecp256k1 0.4.0][] is the latest release of this library for
+  Bitcoin-related cryptographic operations.  The new version includes a
+  module with an implementation of ElligatorSwift encoding; see the
+  project [changelog][libsecp cl] for more information.
+
+- [LND v0.17.0-beta.rc1][] is a release candidate for the next major
+  version of this popular LN node implementation.  A major new
+  experimental feature planned for this release, which could likely
+  benefit from testing, is support for "simple taproot channels".
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #28354][] changes the default value of
+  `-acceptnonstdtxn` to 0 on testnet, which is the default on all
+  other networks. This change may help applications avoid creating
+  transactions that are nonstandard and thus rejected by default nodes
+  on mainnet.
+
+- [LDK #2468][] allows users to provide a `payment_id` which is encrypted in an
+  invoice request's `payer_metadata` field. LDK then looks for that `payment_id`
+  in future invoices and will pay only if it hasn't already paid an invoice for that
```

Comment:
Second sentence I'd say:

"LDK checks the metadata in received invoices and will only pay if it recognizes the id and hasn't already paid another invoice for it."